### DISCUSSION TITLE
开发了一款棋牌游戏，没想好是开源还是继续优化用于商用，但是好像引擎已经无人问津了

### DISCUSSION THREAD

**Discussion by kibozh** [kbengine/kbengine#1591] [General]
引擎现在是什么情况，不再维护了吗？

**Comment by ve-i-uj** [comment]
As far as I know, KBEngine was purchased by Dianhun Network and open source development was discontinued after that. The kbengine forum is closed and the project website is closed. There is only an account on GitHub.

**Reply by kibozh** [reply]
what a puty!

**Reply by charkecc** [reply]
大佬，可以分享学习下吗？

**Comment by qhf566** [comment]
想找一款成熟的手游产品源码过来我们深度二次开发(全新换套美术，代码也都从新货混淆及优化整一遍)。意向最好是重度的mmo、arpg、角色扮演或动作手游，其他类型的游戏也可以，必须是高品质大厂近5年内做的产品，国内厂商或国外厂商开发的都可以，产品不能在国内运营过的。可以在国外运营过或运行中。准备了足够夸张的米给到你。如朋友那边有可以给推荐一下有米，不墨迹,鹅浩809996677
